.circleci/config.yml

```diff
@@ -56,11 +49,10 @@ jobs:
           name: Update golang
           command: |
             sudo rm -rf /usr/local/go/
-            curl -LO https://dl.google.com/go/go1.12.7.linux-amd64.tar.gz
-            sudo tar -C /usr/local -xzf go1.12.7.linux-amd64.tar.gz
+            curl -LO https://dl.google.com/go/go1.13.3.linux-amd64.tar.gz
+            sudo tar -C /usr/local -xzf go1.13.3.linux-amd64.tar.gz
```

Comment:
Why not just `circleci/golang:1.13.3`?

Reply:
This step (`test-integration`) is executed not in the docker image, but on the machine itself (`machine=true`). That's why we install golang manually.

Reply:
Ahh, totally missed that - github concatenated the `machine: true` part :)